### DISCUSSION TITLE
Filtering peak to gene links

### DISCUSSION THREAD

**Discussion by jblich870** [GreenleafLab/ArchR#1830] [General]
Following up on a question raised here https://github.com/GreenleafLab/ArchR/issues/1768

In general what would you suggest is the best way to triage peak 2 gene links to get towards potential regulatory relationships? The number of linked peaks can vary wildly depending on cutoffs used in getPeak2GeneLinks, however, these cutoffs feel fairly arbitrary.

I understand that there is no correct answer here, and that every dataset is likely to be different. I also realise that peak to gene links are correlations and do not directly imply a regulatory relationship. Nonetheless, to my mind the value of identifying such links lies in potential to identify regulatory relationships. I wondered whether you had any thoughts on how best to approach this.

One option might be to set strict correlation, varianceRNA/ATAC and FDR/EmpFDR cutoffs. However, as this is likely to be dataset dependent (e.g. influenced by number of cells and/or peak accessibility/gene expression variance across dataset), perhaps there is another approach to set them in a meaningful way. Just as an example, predicting a priori the rough size of a target gene repertoire and setting cutoffs that identify X number of linked genes.

Identifying peak 2 gene links feels like a potentially very useful analysis but I am still unclear on how to best approach extracting biologically useful information here.

Many thanks,
J

**Comment by rcorces** [comment]
I dont have an answer to this. I have not actually used the permutation-based FDR implementation on my own data. And as you say, this is highly dataset and implementation dependent. I just dont have enough information to draw on to make a recommendation beyond what is described in the manual. It would be great if someone benchmarked this across datasets and implementations but that isnt something that we are doing on our end.

**Reply by jblich870** [reply]
@rcorces Many thanks for the prompt answer. What is your general feeling about the usefulness of peak to gene links? Beyond a purely descriptive metric.

In the spirit of discussion, if anyone else has opinions about this, would be great to hear. (@RegnerM2015 thoughts?)

**Reply by rcorces** [reply]
I think peak to gene links provide some, albeit weak, evidence of a regulatory relationship. P2G links predominantly capture cell type-specific peaks around cell type-specific genes because correlation is very high (peak on, gene on). Given that cell type-specific HiC is still challenging for complex tissues, I think P2G links are an important technique for predicting gene regulatory interactions. But they are only predictions.

**Reply by RegnerM2015** [reply]
I agree that the peak to gene links can provide _some_ evidence of regulatory relationships, but I think it may be helpful to think of the peak to gene correlation analysis as more exploratory or hypothesis generating. There are a variety of approaches to help support the predicted regulatory relationships using outside data sources (including but not limited to eQTL and ENCODE annotations) and experimental validation techniques (including but not limited to CRISPR perturbations, HiC-based techniques, etc.). Therefore, I think the peak to gene association analysis serves as an important step early on in the process of discovering new gene regulatory mechanisms. 

Regarding your questions about the optimal criteria/cutoffs for identifying P2Gs, I also don't have an answer to this. However, this preprint may be helpful (https://www.biorxiv.org/content/10.1101/2022.09.15.507748v1).

**Reply by jblich870** [reply]
@rcorces @RegnerM2015 thanks very much for the thoughtful answers

@RegnerM2015 thanks for the ideas for additional annotations, and the link - it didn't work for me but I assume it is this one: https://europepmc.org/article/ppr/ppr546349

**Reply by rcorces** [reply]
Also relevant though I have not read it yet - https://www.biorxiv.org/content/10.1101/2023.02.01.526609v1

**Reply by RegnerM2015** [reply]
> @rcorces @RegnerM2015 thanks very much for the thoughtful answers
> 
> @RegnerM2015 thanks for the ideas for additional annotations, and the link - it didn't work for me but I assume it is this one: https://europepmc.org/article/ppr/ppr546349

Yes, this is the preprint I was referring to.
